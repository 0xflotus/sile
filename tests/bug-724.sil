\begin[papersize=a5]{document}
\script[src=packages/gutenberg]
\script[src=packages/frametricks]
\script[src=packages/rules]
\showframe[id=all]
\set[parameter=shaper.variablespaces,value=0]
\set[parameter=document.spaceskip,value=8pt plus 16pt]
\nofolios%
\nohyphenation\noindent%
\font[size=30pt]%
\hrule[width=330pt,height=5pt] \alt{a}{bb} \hrule[width=50pt,height=5pt].


\end{document}
