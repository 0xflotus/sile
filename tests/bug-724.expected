Set paper size 	419.5275636	595.275597
Begin page
Draw line	20.9764 29.7638 330.0000 5.0000
Mx 	368.0824
My 	34.7638
Set font 	Gentium Plus;30;400;;normal;;LTR
T	69 69	(bb)
Draw line	20.9764 65.7638 50.0000 5.0000
Mx 	70.9764
My 	70.7638
T	17	(.)
End page
Finish
